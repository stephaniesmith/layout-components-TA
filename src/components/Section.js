import Component from './Component';
// import styles from './Section.css';

export default class Section extends Component {
  render() {
    return `<section>${this.props.message}</section>`;
  }
}
