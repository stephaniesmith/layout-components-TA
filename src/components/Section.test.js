import Section from './Section';

describe('Section Component', () => {
  it('should be a section', () => {
    const section = new Section({
      message: 'Hello World!'
    });

    expect(section.render()).toMatchSnapshot();
  });
});
