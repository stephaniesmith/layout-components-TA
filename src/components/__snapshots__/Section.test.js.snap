// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Component should be a section 1`] = `"<section>Hello World!</section>"`;
